### DISCUSSION TITLE
Copilot icon doesnt show up in a SSH remote access in VSCode

### DISCUSSION THREAD

**Discussion by lxrswdd** [community/community#55087] [Copilot Conversations] [answered]
### Select Topic Area

Question

### Body

Hi, I am accessing a school server using VSCode SSH with a school VPN running.

I have installed Copilot but the icon of Copilot doesn't show up in the bottom right corner of my VSCode. Also, it is not showing any recommendations when I am writing codes. There isn't any error message shown up and Copilot works fine if I am using a local workspace instead of the SSH server.

I have tried adding   
```
 "remote.extensionKind": {
        "GitHub.copilot": ["ui"],
      }
```
to setting.json of Copilot but this doesn't work either.

Can anyone help?

**Comment by Akash1134** [comment]
Hi All,

If you're running version 1.83 for Copilot, I can confirm that Copilot v1.83 might not work with certain VSCode extension when a SSH FS is being used. Our team is looking into it.

Rolling back Copilot to version Rolling back to 1.82.x is currently a workaround but we are working towards a fix.

Thanks!

**Comment by lxrswdd** [comment, marked as answer]
@Akash1134 
Thank you for responding, I solved it. It was because I opened the workspace at the root directory of my system, which contained too many files to be processed, so the Pylance could not activate properly, so did the Copilot. Change the workspace to a folder with few files work.

**Comment by davidoliver-1** [comment]
If the Copilot icon is not showing up in a SSH remote access in VSCode, there are a few things you can try to resolve the issue:

Ensure that you have installed the Copilot extension on the remote machine: The Copilot extension needs to be installed on the remote machine as well as on your local machine in order for it to work properly. Check that the Copilot extension is installed on the remote machine, and if it isn't, install it there.
Verify that the SSH remote is properly configured: Ensure that the SSH remote connection is properly configured in VSCode. Verify that the remote machine is connected to the internet, and that your SSH connection settings are correct.
Check for updates to VSCode and the Copilot extension: Make sure that you are running the latest version of VSCode and the Copilot extension. Check for updates in the Extensions view of VSCode, and update if necessary.
